--- origsrc/kwave-0.8.5/libgui/KwaveFileDialog.cpp	2009-12-24 06:43:36.000000000 -0600
+++ src/kwave-0.8.5/libgui/KwaveFileDialog.cpp	2011-02-09 20:37:41.423833400 -0600
@@ -34,7 +34,7 @@ KwaveFileDialog::KwaveFileDialog(const Q
     const QString &filter, QWidget *parent, bool modal,
     const QString last_url, const QString last_ext)
     :KFileDialog(startDir, filter, parent),
-     m_config_group(0), m_last_url(last_url), m_last_ext(last_ext)
+     m_config_group(QString::null), m_last_url(last_url), m_last_ext(last_ext)
 {
     setModal(modal);
 
--- origsrc/kwave-0.8.5/libgui/KwaveFileDialog.h	2009-12-24 06:43:36.000000000 -0600
+++ src/kwave-0.8.5/libgui/KwaveFileDialog.h	2011-02-09 21:06:02.241196100 -0600
@@ -43,8 +43,8 @@ public:
      */
     KwaveFileDialog(const QString& startDir, const QString& filter,
                     QWidget *parent, bool modal,
-                    const QString last_url = 0,
-                    const QString last_ext = 0);
+                    const QString last_url = QString::null,
+                    const QString last_ext = QString::null);
 
     /** Destructor */
     virtual ~KwaveFileDialog()
--- origsrc/kwave-0.8.5/libgui/MenuItem.cpp	2009-12-24 06:43:36.000000000 -0600
+++ src/kwave-0.8.5/libgui/MenuItem.cpp	2011-02-09 20:40:34.927336000 -0600
@@ -38,7 +38,7 @@ MenuItem::MenuItem(MenuNode *parent,
                    const QKeySequence &shortcut,
                    const QString &uid)
     :MenuNode(parent, name, command, shortcut, uid),
-     m_exclusive_group(0), m_action(0)
+     m_exclusive_group(QString::null), m_action(0)
 {
     Q_ASSERT(parent);
     if (!parent) return;
--- origsrc/kwave-0.8.5/plugins/sonagram/SonagramDialog.cpp	2009-12-24 06:35:18.000000000 -0600
+++ src/kwave-0.8.5/plugins/sonagram/SonagramDialog.cpp	2011-02-09 21:09:09.129951400 -0600
@@ -105,7 +105,7 @@ void SonagramDialog::parameters(QStringL
     list.clear();
 
     // parameter #0: number of fft points
-    param = pointbox ? pointbox->currentText() : QString(0);
+    param = pointbox ? pointbox->currentText() : QString(QString::null);
     list.append(param);
 
     // parameter #1: index of the window function
