--- origsrc/kwave-0.8.8/cmake/KwaveSysinfo.cmake	2012-05-19 14:47:19.000000000 -0500
+++ src/kwave-0.8.8/cmake/KwaveSysinfo.cmake	2012-05-20 18:34:29.182249800 -0500
@@ -19,7 +19,6 @@ SET(_try_src_dir "${CMAKE_BINARY_DIR}${C
 SET(_try_sysinfo "${_try_src_dir}/cmake_try_sysinfo.c")
 WRITE_FILE("${_try_sysinfo}" "
     #include <sys/sysinfo.h>
-    #include <linux/kernel.h>
     int main()
     {
         struct sysinfo info;
--- origsrc/kwave-0.8.8/libkwave/MemoryManager.cpp	2012-05-19 14:47:19.000000000 -0500
+++ src/kwave-0.8.8/libkwave/MemoryManager.cpp	2012-05-20 18:39:23.102061000 -0500
@@ -27,7 +27,6 @@
 #include <QString>
 
 #ifdef HAVE_SYSINFO
-#include <linux/kernel.h> // for struct sysinfo
 #include <sys/sysinfo.h>  // for sysinfo()
 #endif
 
