--- origsrc/kwave-0.8.5/libkwave/Signal.h	2009-12-24 06:42:15.000000000 -0600
+++ src/kwave-0.8.5/libkwave/Signal.h	2011-02-09 19:43:36.800156100 -0600
@@ -14,8 +14,8 @@
  *   (at your option) any later version.                                   *
  *                                                                         *
  ***************************************************************************/
-#ifndef _SIGNAL_H_
-#define _SIGNAL_H_
+#ifndef _LIBKWAVE_SIGNAL_H_
+#define _LIBKWAVE_SIGNAL_H_
 
 #define PROGRESS_SIZE (512 * 3 * 5)
 
@@ -277,4 +277,4 @@ private:
 };
 
 //**********************************************************************
-#endif  /* _SIGNAL_H_ */
+#endif  /* _LIBKWAVE_SIGNAL_H_ */
